_docs/setup/kubernetes/helm.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Istio Helm Chart Instructions
+overview: Instructions for the setup and configuration of Istio using the Helm package manager.
+
+order: 10
```

Comment:
order should not be the same as quickstart

can you attach a preview of the toc/nav once fixed

Reply:
good catch!  Taught me something new.